lib/conditionals.md

```diff
@@ -5,8 +5,33 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+A conditional asks a question to determine the state of being (true or false). 
+It is used in a statement that will result in a specific output depending on the input. 
 
+    Example: If I am hungry, then I will eat food, otherwise I will not eat food.
+
+The 'am' acts as the conditional '=='. 
+The 'if', 'then', and 'otherwise(else)' are the instructions. 
+Whether I am hungry or I am in any other state besides hungry is the input. 
+The output is either that I will eat food or I will not eat food.
+
+ 
 # What is the difference between `=` and `==`?
 
-Replace me with your response...
\ No newline at end of file
+The difference is that "==" is a question and "=" is an assignment.
+
+"==" is used to determine whether or not something is true. Is beth hungry?
+
+    if beth == hungry
```

Comment:
You write that "`==` is used to determine whether or not something is true". But `>`, `<`, and `<=` also check that something is true, in a sense:

`5 > 1` ... `true`
`5 < 1` ... `false`
`5 <= 1` ... `false`
`5 == 1` ... `false`

So you're not wrong in your statement, but there are other places where we use the words "true" and "false", and it can later be confusing if you think of `==` as "determining whether or not something is true".

Can you write a new definition that is more specific about what `==` does?

And with your new definition, reconsider your example: `if beth == hungry`. We would not be likely to use `==` to check if Beth is hungry. Based on your new definition, try to explain why not.